node/node_genesis.go

```diff
@@ -86,9 +88,14 @@ func (node *Node) SetupGenesisBlock(db ethdb.Database, shardID uint32, myShardSt
 		}
 	case nodeconfig.Pangaea:
 		chainConfig = *params.PangaeaChainConfig
-		fallthrough // the rest is the same as testnet
+	case nodeconfig.Partner:
```

Comment:
why don't you add nodeconfig.Testnet here as well?

Reply:
Testnet is the default config for all of the other chain configs.